openpathsampling/tools.py

```diff
@@ -205,24 +205,20 @@ def units_from_snapshot(snapshot):
         the simtk.unit.Unit to be used
     """
 
-    units = {}
+    units = {'velocity': None, 'energy': None, 'length': None}
```

Comment:
This assumes (as it did before) that your snapshots have a concept of velocity, energy, and length. Is there a way to avoid that assumption? Or do we just make that assumption, and then not use those units if they don't describe snapshot features?